src/Solo.ts

```diff
@@ -116,7 +116,7 @@ export class Solo {
       options.wsOrigin,
     );
     this.standardActions = new StandardActions(this.operation, this.contracts);
-    this.walletLogin = new WalletLogin(this.web3, networkId);
+    this.signOffChainAction = new SignOffChainAction(this.web3, networkId);
```

Comment:
You probably want to include a class `WalletLogin` that inherits from `SignOffChainAction` so that you don't introduce breaking changes with this PR. Then over time, get all the clients to migrate over to using `SignOffChainAction` instead, and later delete `WalletLogin` as a breaking change